### PR TITLE
via: Address validation errors on macOS

### PR DESCRIPTION
**Pull request by davidlunarg**
Added portability extensions to vkCreateDevice and vkCreateInstance
calls so that when via is run from vkconfig, validation errors
are not generated.

Addresses issue https://github.com/LunarG/VulkanTools/issues/1495

### CODE REVIEW

File: via/via_system.cpp [hunk 1]
```diff
@@ -459,13 +459,21 @@ ViaSystem::ViaResults ViaSystem::GenerateInstanceInfo(void) {
...
         if (strcmp(extension.extensionName, VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME) == 0) {
+            portability_instance_extension_list.push_back(VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME);
+            inst_info.ppEnabledExtensionNames = portability_instance_extension_list.data();
             inst_info.flags = VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
-            inst_info.enabledExtensionCount = 1;
-            inst_info.ppEnabledExtensionNames = portability_instance_extension_list;
-            break;
+            inst_info.enabledExtensionCount++;
+        }
+        if (strcmp(extension.extensionName, VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME) == 0) {
```
Comment (MarkY-LunarG) [inline via/via_system.cpp:473]: Why did you add device properties 2 here?  I thought this change focused on portability?

Comment (davidlunarg) [reply via/via_system.cpp:473]: Maybe a comment might be in order here explaining why...

On Mac, if you specify VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME without specifying VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME, validation produces an error.

Comment (charles-lunarg) [reply via/via_system.cpp:473]: Not `VK_KHR_portability_enumeration` specifically, but `VK_KHR_portability_subset`  (as the latter is a device extension and needs `vkGetPhysicalDeviceProperties2KHR`).

Comment (davidlunarg) [reply via/via_system.cpp:480]: Force-pushed a change adding comments.

**Review by charles-lunarg — APPROVED** [review]
